yamllint/cli.py

```diff
@@ -129,6 +138,9 @@ def show_problems(problems, file, args_format, no_warn):
                 first = False
             print(Format.standard(problem, file))
 
+    if github_group:
```

Comment:
Can we get rid of this variable `github_group` and simply use:
```suggestion
    if not first and args_format == 'github':
```
?